pkg/services/ngalert/notifier/available_channels.go

```diff
@@ -459,6 +459,14 @@ func GetAvailableNotifiers() []*alerting.NotifierPlugin {
 					PropertyName: "url",
 					Secure:       true,
 				},
+				{
+					Label:        "Endpoint URL",
+					Element:      alerting.ElementTypeInput,
+					InputType:    alerting.InputTypeText,
+					Description:  "Optionally provide a custom Slack message API endpoint, default is https://slack.com/api/chat.postMessage",
```

Comment:
Is it worth mentioning in the description that this is only for non-webhook requests? The webhook approach would already contain the correct base URL inside the webhook config itself, and this setting would have no impact.

Reply:
Many of the other settings do this too - once that are either webhook-specific or not mention this in their descriptions.